src/Commands/Bag.php

```diff
@@ -58,16 +75,16 @@ class Bag extends Object
      * $builder->drop(3)->fromDb('AFTER')
      * @var mixed
      */
-    public $return = false;
+    public $return = null;
```

Comment:
Can we remove this `Bag::$return` property? It shouldn't be needed anymore since the projections are stored in `Bag::$retrieve`